lib/jellyfish_web/api_spec/component/rtsp.ex

```diff
@@ -13,7 +13,32 @@ defmodule JellyfishWeb.ApiSpec.Component.RTSP do
       title: "ComponentPropertiesRTSP",
       description: "Properties specific to the RTSP component",
       type: :object,
-      properties: %{}
+      properties: %{
+        sourceUri: %Schema{
+          type: :string,
+          description: "URI of RTSP source stream",
+          example: "rtsp://localhost:554/stream"
```

Comment:
I wouldn't add an example if this schema is only used to describe a response message. What do you think?